test/26_air_beefmaker_extfn/test.cpp

```diff
@@ -1,203 +1,103 @@
-//===- test.cpp -------------------------------------------------*- C++ -*-===//
+//===- main.cpp -------------------------------------------------*- C++ -*-===//
```

Comment:
the file is test.cpp

Reply:
Good catch. Addressed in [f67896e](https://github.com/Xilinx/mlir-air/pull/266/commits/f67896e1d6f1ff6dfccad61e562540cce8725c98).